src/main/kotlin/com/bardsoftware/papeeria/backend/cosmas/CosmasServer.kt

```diff
@@ -0,0 +1,76 @@
+/**
+Copyright 2018 BarD Software s.r.o
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+ */
+
+package com.bardsoftware.papeeria.backend.cosmas
+
+import io.grpc.stub.StreamObserver
+import com.bardsoftware.papeeria.backend.cosmas.CosmasProto.*
+import com.bardsoftware.papeeria.backend.cosmas.CosmasGrpc.*
+import io.grpc.Server
+import io.grpc.ServerBuilder
+
+/**
+ * Simple server that will wait for request and will send response back
+ * @author Aleksandr Fedotov (iisuslik43)
+ */
+class CosmasServer {
+    private val PORT = 50051

```

Comment:
Please make it configurable via command-line arguments. There is a choice of command line arg parsers, you can take e.g. this one:
https://github.com/xenomachina/kotlin-argparser

Flag name is `--port`.

Reply:
Done